### PR TITLE
schedule push for registered users with empty stories

### LINKED ISSUE #405: send pushes to registered, but non-onboarded users (haven’t submit profile with a story)

### PR DESCRIPTION
**Pull request by akhrail1996**
closes #405 

- [x] manually send push to all current users with empty stories

### CODE REVIEW

File: lib/t/accounts.ex [hunk 1]
```diff
@@ -682,4 +684,24 @@ defmodule T.Accounts do
...
   end
+
+  def push_users_to_complete_onboarding do
+    start_date = DateTime.add(DateTime.utc_now(), -24 * 60 * 60)
+    finish_date = DateTime.add(DateTime.utc_now(), -24 * 60 * 60 - 60)
+
+    Profile
+    |> where([p], is_nil(p.story))
+    |> where([p], p.last_active <= ^start_date)
+    |> where([p], p.last_active >= ^finish_date)
```
Comment (akhrail1996) [inline lib/t/accounts.ex:695]: didn't figure how to use SQL fragment now() with correct timezone

**Comment by ruslandoga** [comment]
I don't understand the goal here

File: lib/t/accounts.ex (hunk 1 above)
Comment (ruslandoga) [reply lib/t/accounts.ex:695]: what timezone is correct?

Comment (akhrail1996) [reply lib/t/accounts.ex:695]: UTC?

Comment (ruslandoga) [reply lib/t/accounts.ex:695]:
```sql
select now();
-- 2021-11-16 11:25:40.825731+00
```

```elixir
iex(1)> DateTime.utc_now 
~U[2021-11-16 11:25:48.669111Z]
```

**Comment by akhrail1996** [comment]
The goal is to "activate" users which started profile creation, but switched to other things / forgot about it at some point and never finished their stories -> never actually tried the app

surprisingly there is a lot of such users rn (181 / 530)

File: lib/t/accounts.ex (hunk 1 above)
Comment (ruslandoga) [reply lib/t/accounts.ex:695]: https://xyproblem.info

**Comment by ruslandoga** [comment]
how does it work?

File: lib/t/accounts.ex (hunk 1 above)
Comment (akhrail1996) [reply lib/t/accounts.ex:695]: thanks, the solution to this indeed turned out to be:
`psql`
`ALTER DATABASE t_test SET timezone TO 'UTC';`

Comment (akhrail1996) [reply lib/t/accounts.ex:695]: @orlovvlad95 once I merge this PR into master you will experience this issue as well when testing, so pls run the above commands in your shell

**Pull request merged**